index.html

```diff
@@ -113,13 +113,13 @@
     //
     //-------------------------------------------------------------------------
 
```

Comment:
The black and whit colors could cause a problem with visualization, maybe try to swap those 2 colors.